### PR TITLE
Updated files localization and added message if user don't have perms

### PR DESCRIPTION
**Pull request by LWJerri**

### CODE REVIEW

**Comment by Satont** [comment]
Hi! Thank you for pool request.

Instead of doing 
```ts
if(!msg.member.permissions.has('MANAGE_CHANNELS')){
 return msg.reply(msg.guild.lang.get('errors.noPermissions'))
}
```
Every time you can move it to 
https://github.com/Satont/channelsbot/blob/master/src/index.ts#L51-L52

```ts
if (typeof command.checkCustomPerm !== 'undefined' && !command.checkCustomPerm(msg))  return msg.reply(msg.guild.lang.get('errors.noPermissions'))
if (typeof command.permission !== 'undefined' && !msg.member.hasPermission(command.permission))  return msg.reply(msg.guild.lang.get('errors.noPermissions'))
```

**Comment by Satont** [comment]
Thats why i added `permission` property and `checkCustomPerm`.

This way more flexible.

**Comment by LWJerri** [comment]
Okay. Today I make this.

File: src/commands/lang.ts [hunk 1]
```diff
@@ -5,11 +5,13 @@ import db from '../db'
...
-  checkCustomPerm(msg) {
-    return (msg.member.hasPermission('ADMINISTRATOR') || msg.member.hasPermission('MANAGE_GUILD'))
-  },
   async run(msg, args) {
+    if(!msg.member.hasPermission('MANAGE_GUILD') || !msg.member.hasPermission('ADMINISTRATOR')){
+      return msg.reply(msg.guild.lang.get('errors.noPermissions'))
+    }
+    
     const lang = args[0]
+    if (lang == undefined) return msg.reply(msg.guild.lang.get('errors.noLang', lang))
```
Comment (Satont) [inline src/commands/lang.ts:14]: Thats unnecessary. 

```js
lang !== 'russian' && lang !== 'english'
```

Doing that work for us. 

example: `undefined !== 'russian'` will be `true`

File: langs/russian.json [hunk 2]
```diff
@@ -47,6 +47,8 @@
     }
   },
   "errors": {
-    "langStringNotFound": "Перевод для данной строки не найден."
+    "langStringNotFound": "Перевод для данной строки не найден.",
+    "noPermissions": "у тебя нету права `MANAGE_GUILD` или `ADMINISTRATOR`, чтобы использовать данную команду!",
```
Comment (Satont) [inline langs/russian.json:51]: "нет", а не "нету".

**Pull request merged**